src/barChart.ts

```diff
@@ -1,124 +1,121 @@
-import "./../style/visual.less";
 import {
+    BaseType,
     select as d3Select,
-    Selection as d3Selection,
-    BaseType
+    Selection as d3Selection
 } from "d3-selection";
 import {
+    scaleBand,
     scaleLinear,
-    ScaleLinear,
-    scaleBand
+    ScaleLinear
 } from "d3-scale";
 
 import { axisBottom } from "d3-axis";
 
 import powerbiVisualsApi from "powerbi-visuals-api";
+import { createTooltipServiceWrapper, ITooltipServiceWrapper } from "powerbi-visuals-utils-tooltiputils";
+import { FormattingSettingsService } from "powerbi-visuals-utils-formattingmodel";
+import { textMeasurementService, valueFormatter } from "powerbi-visuals-utils-formattingutils";
+import { dataViewWildcard } from "powerbi-visuals-utils-dataviewutils";
+import { HtmlSubSelectableClass, HtmlSubSelectionHelper, SubSelectableDirectEdit as SubSelectableDirectEditAttr, SubSelectableDisplayNameAttribute, SubSelectableObjectNameAttribute, SubSelectableTypeAttribute } from '../node_modules/powerbi-visuals-utils-onobjectformatting/src';
```

Comment:
You are going to remove this  "powerbi-visuals-utils-onobjectformatting" npm package .
Where will these objects come from then?

Reply:
powerbi-visual-utils-onobjectutils package